src/components/ComputeModal.js

```diff
@@ -101,6 +93,34 @@ const DataprocDiskSelector = ({ value, onChange }) => {
   ])
 }
 
+const SparkInterface = ({ interfaceName, synopsis, namespace, name, onDismiss }) => {
+  const interfaceDisplayName = Utils.switchCase(interfaceName,
+    ['yarn', () => 'YARN Resource Manager'],
+    ['apphistory', () => 'YARN Application Timeline'],
+    ['hdfs', () => 'HDFS NameNode'],
```

Comment:
not sure how we would fix it, but the double sameish-color-but-slightly-off header is really jarring for hdfs

![image](https://user-images.githubusercontent.com/6465084/135642509-6052639e-76b5-4c4f-aa4c-9b52f6fda8d6.png)

Reply:
@kyuksel  is this green header from hadoop branding?

Reply:
Yea it's from the particular Spark UI. I was going to bring this up during the UX meeting scheduled for 4pm today.

Reply:
The outcome was to take this particular UI out and consider adding it back in with a differenting styling. The details are in the [respective ticket](https://broadworkbench.atlassian.net/browse/IA-2985).